CHANGELOG.rst

```diff
@@ -4,8 +4,17 @@ lib_xua change log
 UNRELEASED
 ----------
 
-  * FIXED: Issue with MCLK not present for digital RX only configs when using the software PLL
-  * FIXED: Reset SW PLL phase/frequency detector when digital clock becomes invalid to prevent incorrect error input to sigma-delta modulator
+  * ADDED: Support for optional include header `user_main_tasks.h` with the same
+    functionality as `user_main_cores.h` i.e. to allow insertion of tasks into
```

Comment:
I have never seen a user_main_cores.h. Do you mean user_main.h?

Reply:
It's an optional header file (like xua_conf.h) just we never used it before 

See: https://github.com/xmos/lib_xua/blob/1b87f36453fac4164700a13d1771a68fbb94204f/lib_xua/src/core/main.xc#L486